configs/wider_face/ssd300_wider_face.py

```diff
@@ -1,20 +1,29 @@
 _base_ = [
```

Comment:
how about add `faster_rcnn_r50_fpn_1x config`and `retinanet_r50_fpn_1x`?

Reply:
we can do that in the future. For now, we can fix existing issues.